docs/pages/management/guides/docker.mdx

```diff
@@ -11,7 +11,9 @@ images and execute commands on that container via Teleport's `tsh` client.
 
 Since all of Teleport's services are run from the same binary, you can
 use our Docker image to run Node services (e.g., the Database Service or App
-Service) or explore the Auth and Proxy Services locally.
+Service) or explore the Auth and Proxy Services locally. The distroless container
```

Comment:
If we're expanding this Notice to introduce the scope of this guide, we should make it visible to all scopes.

Also, the let's not say "Node services", since the docs don't use this term anymore.

Finally, will a reader know what "the distroless container" means at this point on the page? I think we need to introduce the fact that Teleport containers are distroless by default before we say that we are joining a container to the cluster to demonstrate server access.

Reply:
Please see updates @ptgott , distroless address and updated scope for this